src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello world</title>
+</head>
+<body>
+    Hello, world!
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags.
https://mate-academy.github.io/style-guides/htmlcss.html#indentation